test/location.coffee

```diff
@@ -436,6 +436,9 @@ test "#3822: Simple string/regex start/end should include delimiters", ->
   eq regexToken[2].last_line, 0
   eq regexToken[2].last_column, 6
 
+strVal = (str) ->
+  str.valueOf?() ? str
```

Comment:
Cleanup

Reply:
@GeoffreyBooth this one is actually getting used (see below), since `eq()` was failing when comparing a primitive string to a `String()` object

It is equivalent to the `unwrapStringObject()` (now `normalizeStringObject()`) helper though, I don't know if there's an existing way to share a helper between `src/` and tests?